src/components/Header/index.js

```diff
@@ -142,4 +141,9 @@ const Header = ({
   );
 };
 
-export default compose(injectState, withTheme, withRouter, withApi)(Header);
+export default compose(
+  injectState,
+  withTheme,
+  withRouter,
+  withApi,
+)(Header);
```

Comment:
I'm guessing everything here is just prettier... can you confirm @ciaranschutte? :)

Reply:
Confirmed :)